docs/snippets/providers/ollama-snippet-autogenerated.mdx

```diff
@@ -0,0 +1,31 @@
+{/* This snippet is automatically generated using scripts/docs_render_provider_snippets.py 
+Do not edit it manually, as it will be overwritten */}
+
+## Authentication
+This provider requires authentication.
+- **host**: Ollama API Host URL (required: True, sensitive: False)
+
+
+## In workflows
+
+This provider can be used in workflows.
+
+
+As "step" to query data, example:
+```yaml
+steps:
+    - name: Query ollama
+      provider: ollama
+      config: "{{ provider.my_provider_name }}"
+      with:
+        prompt: {value}  
+        model: {value}  
+        max_tokens: {value}  
+        structured_output_format: {value}  
+```
+
+
+
+
+
+If you need workflow examples with this provider, please raise a [GitHub issue](https://github.com/keephq/keep/issues).
```

Comment:
maybe create a component for that? :D so if we want to edit it one day

Reply:
If we want to edit, we could re-generate them using Jinja template